components/LargeHeader.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+function LargeHeader() {
+  return (
+    <nav className="navbar navbar-default largeheader_background" style={{backgroundImage: 'url(' + 'assets/images/bg.png' +')'}}>
```

Comment:
need to change this so that its loaded properly in CSS with urlloader or fileloader (havent worked that out)